arangod/Transaction/Manager.cpp

```diff
@@ -108,7 +108,6 @@ void Manager::registerTransaction(TransactionId transactionId,
   if (!isReadOnlyTransaction && !isFollowerTransaction) {
     LOG_TOPIC("ccdea", TRACE, Logger::TRANSACTIONS)
         << "Acquiring read lock for tid " << transactionId.id();
-    _rwLock.lockRead();
     _nrReadLocked.fetch_add(1, std::memory_order_relaxed);
```

Comment:
Only slightly related but may be addressed anyway by this PR:
`_nrReadLocked` is only counted up and down, but serves no other purpose than its value being reported in `TRACE` log messages. I guess that this variable was originally added to debug problems during hot backup (and its deadlocks). I would consider it superfluous by now, so IMHO we can as well remove it.
But I am happy if the variable is kept for now if it is still considered useful.

Reply:
Gone.